avro/src/main/java/com/fasterxml/jackson/dataformat/avro/schema/ArrayVisitor.java

```diff
@@ -1,16 +1,15 @@
 package com.fasterxml.jackson.dataformat.avro.schema;
 
-import java.util.List;
-
-import org.apache.avro.Schema;
-import org.apache.avro.Schema.Type;
-
 import com.fasterxml.jackson.databind.JavaType;
 import com.fasterxml.jackson.databind.JsonMappingException;
 import com.fasterxml.jackson.databind.SerializerProvider;
 import com.fasterxml.jackson.databind.jsonFormatVisitors.JsonArrayFormatVisitor;
 import com.fasterxml.jackson.databind.jsonFormatVisitors.JsonFormatTypes;
 import com.fasterxml.jackson.databind.jsonFormatVisitors.JsonFormatVisitable;
+import org.apache.avro.Schema;
```

Comment:
Please do not reorder import statements automatically in future... I use ordering from JDK types to non-Jackson types to Jackson types on purpose.

Reply:
Right - reordering is undone.